google/cloud/security/common/data_access/violation_format.py

```diff
@@ -154,34 +46,23 @@ def format_cloudsql_acl_violation(violation):
     rule_name = violation.rule_name
     if rule_name:
         rule_name = rule_name[:255]
-
-    instance_name = violation.instance_name
-    if instance_name:
-        instance_name = instance_name[:255]
-
-    authorized_networks = str(violation.authorized_networks)
-    if authorized_networks:
-        authorized_networks = authorized_networks[:255]
-
-    ssl_enabled = str(violation.ssl_enabled)
-
     yield (resource_type,
            resource_id,
            rule_name,
            violation.rule_index,
            violation.violation_type,
-           instance_name,
-           authorized_networks,
-           ssl_enabled)
+           json.dumps(violation.violation_data))
 
 def format_groups_violation(violation):
     """Format the groups violation data into a tuple.
 
     Args:
-        violation: The groups violation.
+        violation (namedtuple): The groups violation. This is a named tuple
+            with the following attributes 'member_email','parent.member_email',
```

Comment:
It might be worth defining these violation named tuples elsewhere. Feel free to leave this as a TODO, but having a central type of these violation definitions could be useful.